resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/TestConstants.scala

```diff
@@ -16,7 +16,7 @@
  */
 package org.apache.spark.deploy.k8s.integrationtest
 
-package object constants {
+object TestConstants {
```

Comment:
I removed the package objects and replaced them with normal ones. The package statement was wrong and their naming convention was not following scala rules.